src/log.rs

```diff
@@ -515,7 +511,7 @@ mod tests {
             .stream()
             .boxed()
             .next()
-            .immediate()
+            .wait()
```

Comment:
Ah OK, it appears to be related to the log stream! I think its not really an important issue then.